stack/app/variables.tf

```diff
@@ -76,9 +76,9 @@ variable "public_buckets_list" {
     bucket_name                     = string
     versioning                      = bool
     primary_storage_class_retention = number
-    replicas = list(object({
+    replicas = optional(list(object({
       bucket_arn = string
-    }))
+    })))
```

Comment:
same here, if that works

Reply:
Works 🎉 
Thanks for the recommendations!